qiita_db/test/test_user.py

```diff
@@ -353,6 +361,53 @@ def test_change_forgot_password_bad_code(self):
         self.assertEqual(obsbool, False)
         self._check_pass("password")
 
+    def test_messages(self):
+        user = User('test@foo.bar')
+        obs = user.messages()
+        exp_msg = [
+            (1, 'message 1'),
+            (2, 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. '
+                'Pellentesque sed auctor ex, non placerat sapien. Vestibulum '
+                'vestibulum massa ut sapien condimentum, cursus consequat diam'
+                ' sodales. Nulla aliquam arcu ut massa auctor, et vehicula '
+                'mauris tempor. In lacinia viverra ante quis pellentesque. '
+                'Nunc vel mi accumsan, porttitor eros ut, pharetra elit. Nulla'
+                ' ac nisi quis dui egestas malesuada vitae ut mauris. Morbi '
+                'blandit non nisl a finibus. In erat velit, congue at ipsum '
+                'sit amet, venenatis bibendum sem. Curabitur vel odio sed est '
+                'rutrum rutrum. Quisque efficitur ut purus in ultrices. '
+                'Pellentesque eu auctor justo.'),
+            (3, 'message <a href="#">3</a>')]
+        self.assertEqual([(x[0], x[1]) for x in obs], exp_msg)
+        self.assertTrue(all(x[2] < datetime.now() for x in obs))
+        self.assertFalse(all(x[3] for x in obs))
```

Comment:
You are missing the test for `x[4]` as there are 5 elements on the returned list:
```
In [1]: from qiita_db.user import User

In [2]: u = User('test@foo.bar')

In [3]: u.messages()
Out[3]: 
[[1L,
  'message 1',
  datetime.datetime(2015, 8, 10, 12, 57, 17, 499881),
  False,
  False],
 [2L,
  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque sed auctor ex, non placerat sapien. Vestibulum vestibulum massa ut sapien condimentum, cursus consequat diam sodales. Nulla aliquam arcu ut massa auctor, et vehicula mauris tempor. In lacinia viverra ante quis pellentesque. Nunc vel mi accumsan, porttitor eros ut, pharetra elit. Nulla ac nisi quis dui egestas malesuada vitae ut mauris. Morbi blandit non nisl a finibus. In erat velit, congue at ipsum sit amet, venenatis bibendum sem. Curabitur vel odio sed est rutrum rutrum. Quisque efficitur ut purus in ultrices. Pellentesque eu auctor justo.',
  datetime.datetime(2015, 8, 10, 12, 57, 17, 499881),
  False,
  False],
 [3L,
  'message <a href="#">3</a>',
  datetime.datetime(2015, 8, 10, 12, 57, 17, 499881),
  False,
  False]]
```